@vates/coalesce-calls/index.test.js

```diff
@@ -1,15 +1,18 @@
 'use strict'
 
-/* eslint-env jest */
-
-const { coalesceCalls } = require('./')
+// eslint-disable-next-line n/no-unpublished-require
+const { describe, it } = require('test')
+const assert = require('assert')
+const { coalesceCalls } = require('./index')
```

Comment:
I personally like to separate global from local packages in imports/requires :slightly_smiling_face: 

```suggestion

const { coalesceCalls } = require('./index')
```